styles/index.css

```diff
@@ -18,30 +18,29 @@
     display: flex;
     gap: 70px;
     justify-content: center;
-    text-align: start;
     align-items: center;
     position: absolute;
-    left: 200px;
-    bottom: 70px;
+    left: 100px;
+    top: 30px;
 }
 
 /* İmage Start */
 
-.honeymoon-image img{
-    width: 750px;
-    height: 800px;
+.honeymoon-image .honeymoon-img{
+   width: 570px;
+   position: relative;
+   height: 530px;
 }
 
 /* İmage Finish */
 
 /* Honeymoon Plus Effect Start */
 .honeymoon-plus-effects{
-    position: relative;
+    position: absolute;
     display: flex;
     gap: 20px;
     font-family: 'Roboto', sans-serif;
-    left: 636px;
-    bottom: -5px;
+    left: 458px;
```

Comment:
bu kadar büyük piksel yerine right kullanabilirsin

Reply:
![uyarı 2](https://github.com/archis-academy/tour-and-travel-dec-3-2023-tr/assets/153675498/0e312f93-9f9a-4e19-aceb-a7aa0a85acdc)

right kullanırsam , bu şekilde düzenleme yapabiliyorum.. right : 0; olunca, +'lar ekranın biraz sol tarafında kalıyor.

Reply:
tutundukları relative olan div de bir problem olabilir o zaman onu kontrol edebilirsin